packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -1,8 +1,43 @@
 <template>
     <div
         :class="$style['c-imageTile']"
-        data-test-id="imageTile">
-        I am a ImageTile Component (GB)
+        data-test-id="image-tile-component">
+        <a
+            :class="[
+                $style['c-imageTile-link'], {
+                    [$style['c-imageTile-link--toggle']]: !isLink
+                }]"
+            :href="isLink ? href : '#'"
+            :aria-hidden="isLink ? false : true"
+            :tabindex="isLink ? false : -1"
```

Comment:
according to documentation the tabindex attribute, if specified, must have a value that is a valid integer https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex. Should we use "0" instead of "false"?

Reply:
When `isLink` is true I don't want the `tabindex` property to be output.
When `isLink` is false I want `tabindex` to be `-1`

This is working for me but agree if we wanted to specify a value for tabindex when `isLink` is true `0` would be a more appropriate value

Reply:
A value of -1 will remove the item from the normal tabbing sequence: https://dequeuniversity.com/rules/axe/3.0/tabindex
So this is the correct behaviour.
Also, an integer is a positive or negative whole number, including 0.